### PR TITLE
Support varying value lengths from run to run

### PR DESCRIPTION
**Pull request by beckerg**
Signed-off-by: Greg Becker <gbecker@micron.com>

### CODE REVIEW

**Comment by beckerg** [comment]
This patch fixes issues with varying the value length from run to run.

File: tools/kmt/kmt.c [hunk 1]
```diff
@@ -1348,18 +1416,20 @@ chk_update(struct km_impl *impl, struct km_rec *r, bool reset_hash)
         hash_update(r);
 
     /* If you get a SIGBUS here during init ensure you have
-     * at least 16G of free space in /var/tmp.
+     * less than 16G of free space in /var/tmp.
```
Comment (alexttx) [inline tools/kmt/kmt.c:1419]: Confused.  Made more sense before: SIGBUS if there's not enough space

**Review by alexttx — APPROVED** [review]

File: tools/kmt/kmt.c (hunk 1 above)
Comment (beckerg) [reply tools/kmt/kmt.c:1419]: Doh!  I read it wrong...

**Review by alexttx — APPROVED** [review]

File: tools/kmt/kmt.c (hunk 1 above)
Comment (beckerg) [reply tools/kmt/kmt.c]: Unfixed :-)

**Pull request merged**